src/scripts/controllers/editClientDetailsController.ts

```diff
@@ -63,4 +63,65 @@ export async function postEditClientName(req: Request, res: Response, next: Next
   } catch (error) {
     next(error);
   }
+}
+
+/**
+ * Renders the edit client email address form for a given case reference.
+ * @param {Request} req - Express request object
+ * @param {Response} res - Express response object
+ * @param {NextFunction} next - Express next middleware function
+ */
+export async function getEditClientEmailAddress(req: Request, res: Response, next: NextFunction): Promise<void> {
+  const caseReference = safeString(req.params.caseReference);
+  try {
+    const response = await apiService.getClientDetails(req.axiosMiddleware, caseReference);
+    let currentEmail = '';
+    if (response.status === 'success' && response.data !== null) {
```

Comment:
may be a check to see `response.data?.emailAddress` ?

Reply:
Have amended to;

>     if (response.status === 'success' && response.data?.emailAddress) {
>       currentEmail = safeString(response.data.emailAddress);
>     } 

Agree with the rationale to have an extra check that we get a some response data back and the theire is `emailAddress` in that response data.